reinforcement-learning-games/zombsole-gym-example/dqn/layer.py

```diff
@@ -0,0 +1,77 @@
+'''
```

Comment:
This file might not be used.  Also, it appears `layers.py` was either intended to replace this file, or this file was meant as an experiment.